torchao/prototype/quantized_training/int8.py

```diff
@@ -12,13 +12,41 @@
 _c10d_functional = torch.ops._c10d_functional
 
 
+@torch.no_grad()
+def quantize_int8_rowwise(tensor: Tensor, stochastic_rounding: bool = False):
+    """Normal rounding will always round down small changes in weight update. To tackle this problem,
+    stochastic rounding can be used, which has a low chance, but not zero, of rounding up. The
+    probability of rounding up is equal to x - ⌊x⌋, which indicates how close the value is to the next
+    integer value. Thus, stochastic rounding also approximates the floating point value exactly.
```

Comment:
add a note saying stochastic rounding seems to be not needed for int8 mixed precision?